app/helpers/posts_helper.rb

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Do we need this file? Or should we move the helper method form the posts controller here?